modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js

```diff
@@ -63,8 +63,15 @@ const FrequentMutationsChartComponent = compose(
             tooltipHTML="Download image or data"
           />
         </div>
-        <div style={graphTitle()}>Distribution of Most Frequent Mutations</div>
-        <Row id="mutation-chart" style={{ padding: '10px', justifyContent: 'center' }}>
+        <div
+          style={{
+            ...graphTitle(),
+            width: showSurvivalPlot ? width : ((width / 2) - chartMargin.left - chartMargin.right),
```

Comment:
this is a bit strange, and also leaves the download button way over to right.
It might be better to have a parent container for the barchart/button/title that either has it's width set or is `display: inline-block`